config.py

```diff
@@ -15,3 +15,8 @@ class Config:
 
     DATABASE_SCHEMA = os.getenv("DATABASE_SCHEMA", "securemessage")
     DATABASE_URI = os.getenv("DATABASE_URI", "postgresql://postgres:postgres@localhost:5432/postgres")
+    JWT_KEY = os.getenv("JWT_KEY", "test-key")
+    UAA_URL = os.getenv("UAA_URL", "http://localhost:9080")
```

Comment:
In order for this to work without separately configuring the URL in the config for each environment, we should additional use the Kubernetes service host/port injection in the deployment helm template. This should ensure the `UAA_SERVICE_HOST` and `UAA_SERVICE_PORT` (provided by Kubernetes service registration) are used without having to hardcode.

Reply:
Kind of my point above, the UAA has more meat on the bone to work out. I didn't want to drag the story out as it felt like an independent piece of work

Reply:
I think the central purpose of this flow is to authenticate the token by checking the signature is valid against UAA (by executing a GET request on the token). It's an alternative authorisation to using basic auth (and has the benefit of giving us scopes for that user). The integration with UAA is central to that which is why it wasn't a separate card.

If UAA is unable to verify the signature it will return a `400 Invalid token to access this Microservice Resource`.

Reply:
You can do it like that, but there is a clear separation in creating a decorator, which is checking a token is valid and has the correct claims and then bringing in the UAA element. It is a 5 SP story after all and works as a standalone piece. There is plenty of work that can be reviewed on this PR